demo/asset_section.js

```diff
@@ -238,8 +281,17 @@ shakaDemo.load = function() {
   // Revert to default poster while we load.
   shakaDemo.localVideo_.poster = shakaDemo.mainPoster_;
 
-  // Load the manifest.
-  player.load(asset.manifestUri).then(function() {
+  var configureCertificate = Promise.resolve();
+
+  if (asset.certificateUri) {
```

Comment:
If you're going to have extra fields on the assets, you should add this field to the `shakaAssets.AssetInfo` type definition in `demo/common/assets.js`.

Reply:
Updated.